contracts/protocols/mainnet/aave_v3/helpers.sol

```diff
@@ -0,0 +1,361 @@
+        return 0x074eE9683639930D305184586B364c8C19ED3d4d; //Rinkeby IncentivesProxyAddress
+    }
+
+    /**
+     *@dev Returns AaveOracle Address
+     */
+    function getAaveOracle() internal pure returns (address) {
+        return 0xc1ee6d09C2A682490cb728aF0E87859E329e1705; //Rinkeby address
+    }
+
+    /**
+     *@dev Returns StableDebtToken Address
+     */
+    function getStableDebtToken() internal pure returns (address) {
+        return 0xE0987FC9EDfcdcA3CB9618510AaF1D565f4960A6; //Rinkeby address
+    }
+
+    function getChainLinkFeed() internal pure returns (address) {
+        return 0x8A753747A1Fa494EC906cE90E9f37563A8AF630e;
+    }
+
+    struct BaseCurrency {
+        uint256 baseUnit;
+        address baseAddress;
+        string symbol;
+    }
+
+    struct EModeCategoryData {
+        uint16 ltv;
+        uint16 liquidationThreshold;
+        uint16 liquidationBonus;
+        uint256[] price;
+        address priceSource;
+        string label;
+    }
+
+    struct AaveV3UserTokenData {
+        uint256 tokenPriceInBase;
+        uint256 supplyBalance;
+        uint256 stableBorrowBalance;
+        uint256 variableBorrowBalance;
+        uint256 supplyRate;
+        uint256 stableBorrowRate;
+        uint256 userStableBorrowRate;
+        uint256 variableBorrowRate;
+        bool isCollateral;
+        uint256 price;
+        flags flag;
+    }
+
+    struct AaveV3UserData {
+        uint256 totalCollateralBase;
+        uint256 totalBorrowsBase;
+        uint256 availableBorrowsBase;
+        uint256 currentLiquidationThreshold;
+        uint256 ltv;
+        uint256 healthFactor;
+        uint256 eModeId;
+        BaseCurrency base;
+        uint256 pendingRewards;
+    }
+
+    struct AaveV3TokenData {
+        uint256 decimals;
+        uint256 ltv;
+        uint256 threshold;
+        uint256 reserveFactor;
+        uint256 totalSupply;
+        uint256 availableLiquidity;
+        uint256 totalStableDebt;
+        uint256 totalVariableDebt;
+        uint256 collateralEmission;
+        uint256 debtEmission;
+        AaveV3Token token;
+    }
+
+    struct flags {
+        bool usageAsCollateralEnabled;
+        bool borrowEnabled;
+        bool stableBorrowEnabled;
+        bool isActive;
+        bool isFrozen;
+    }
+
+    struct AaveV3Token {
+        uint256 supplyCap;
+        uint256 borrowCap;
+        uint256 eModeCategory;
+        uint256 debtCeiling;
+        uint256 debtCeilingDecimals;
+        uint256 liquidationFee;
+        bool isolationBorrowEnabled;
+        bool isPaused;
+    }
+
+    function getTokensPrices(address[] memory tokens)
+        internal
+        view
+        returns (uint256[] memory tokenPrices, uint256 ethPrice)
+    {
+        uint256[] memory tokenPrices = IAaveOracle(getAaveOracle()).getAssetsPrices(tokens);
+        (, int256 EthPrice, , , ) = (AggregatorV3Interface(getChainLinkFeed()).latestRoundData());
+        ethPrice = uint256(EthPrice);
```

Comment:
```suggestion
        uint256[] memory _tokenPrices = IAaveOracle(getAaveOracle()).getAssetsPrices(tokens);
        (, int256 EthPrice, , , ) = (AggregatorV3Interface(getChainLinkFeed()).latestRoundData());
        ethPrice = uint256(EthPrice);
        tokenPrices = new TokenPrice[](_tokenPrices.length);
         for (uint256 i = 0; i < _tokenPrices.length; i++) {
            tokenPrices[i] = TokenPrice(_tokenPrices[i], wmul(_tokenPrices[i], uint256(ethPrice) * 10**10));
        }
```

Reply:
The returned prices are in Base_currency though the units are wei only, can we apply ETH/USD conversion here?